pubspec-nix/pubspec-nix.py

```diff
@@ -192,7 +193,7 @@ def main():
 
     pubspec_lock = yaml.safe_load(open("pubspec.lock", "r"))
 
-    pbar = tqdm(total=len({ k: v for k, v in pubspec_lock["packages"].items() if "url" in v["description"] }))
+    pbar = tqdm(total=len({ k: v for k, v in pubspec_lock["packages"].items() if v["source"] == "git" or v["source"] == "hosted" }))
```

Comment:
it'd be nicer semantically if the program knew about this more generally

Reply:
done